scss/mixins/_background-variant.scss

```diff
@@ -6,10 +6,12 @@
   #{$parent} {
     background-color: $color !important;
   }
-  a#{$parent} {
+  a#{$parent},
+  button#{$parent} {
     @include hover-focus {
       background-color: darken($color, 10%) !important;
     }
+    @include transition($btn-transition);
```

Comment:
Let's ditch the transition—there's no need to match this against buttons because `.bg-` utilities shouldn't be used with our buttons.